### PR TITLE
Test normalize ast

### PR DESCRIPTION
**Pull request by wert-rar**
Delete functhion and todo comment

### CODE REVIEW

File: .idea/inspectionProfiles/Project_Default.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<component name="InspectionProjectProfileManager">
```
Comment (0dminnimda) [inline .idea/inspectionProfiles/Project_Default.xml:1]: Could you please remove these `xml`s'?
You can use `.gitignore` to ignore all files in the .idea directory:
just add the line with `.idea /` to the file

File: tests/test_normalize_ast.py [hunk 2]
```diff
@@ -81,8 +81,7 @@ def test_visit_ExtSlice():
 
 
 ############ normalize_ast #############
-# TODO: add a test that shows that ast.fix_missing_locations is in use,
-# it is probably better to take tests for ast.fix_missing_locations from python sources.
+
```
Comment (0dminnimda) [inline tests/test_normalize_ast.py:84]: What's the point of just deleting this comment?
The `fix_missing_locations` function is still in use in the code, [look](https://github.com/0dminnimda/pynalyser/blob/19784feaf4f0166dbcaef58d058dec2b6d864d95/pynalyser/normalize_ast.py#L34).

Comment (0dminnimda) [reply tests/test_normalize_ast.py:84]: And what is the reason for removing this function if you want to do this?

**Review by 0dminnimda — CHANGES_REQUESTED** [review]

**Comment by wert-rar** [comment]
я сделал то что ты сказал, разве не так ?
--
Отправлено из Mail.ru для Android пятница, 20 августа 2021г., 11:56 +03:00 от 0dminnimda  ***@***.*** :

***@***.*** requested changes on this pull request.

**Comment by 0dminnimda** [comment]
Somehow the test_normalize_ast branch appeared, in order to delete it from this repository, I need to close this PR

**Pull request closed without merging**